rolling/distribution.yaml

```diff
@@ -1565,6 +1565,21 @@ repositories:
       url: https://github.com/eProsima/Fast-DDS.git
       version: 2.14.x
     status: maintained
+  feetech_ros2_driver:
+    doc:
+      type: git
+      url: https://github.com/JafarAbdi/feetech_ros2_driver.git
+      version: main
+    release:
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/JafarAbdi/feetech_ros2_driver-release.git
```

Comment:
Do you want to consider housing the release repo in the ros2-gbp org instead? The advantage there is that your package will automatically be available in the next ROS 2 distribution on release day.
https://docs.ros.org/en/rolling/How-To-Guides/Releasing/First-Time-Release.html#create-a-new-release-repository

Reply:
@Yadunund Do you mind taking a look at https://github.com/ros2-gbp/ros2-gbp-github-org/pull/663? Thanks!